api/src/pcapi/connectors/acceslibre.py

```diff
@@ -23,6 +24,24 @@
 MATCHING_RATIO = 50
 
 
+class AccesLibreApiException(Exception):
+    pass
+
+
+class AccessibilityParsingException(Exception):
+    pass
+
+
+class AccessibilityInfo(pydantic_v1.BaseModel):
+    access_modality: list[str] | None = None
```

Comment:
[discussion] Je suppose qu'on reçoit effectivement une liste ou rien (`null`) de l'API, mais souhaite-t-on pour autant stocker ça tel quel ? Y-a-t-il une différence entre une valeur nulle et une liste vide, pour ces champs ? S'il n'y a pas de différence, je suggère de typer en `list` (au lieu de `list | None`) car : 
- l'usage ensuite est plus simple. Par exemple, pour chercher si un élément est dans la valeur, on peut simplement écrire `if item in values` au lieu de `it item in (values or [])` (ou, beurk,  `if values and item in values`).
- ... et accessoirement, ça évite d'oublier les cas où `value` est `None`. ;)

Bref, je suggère quelque chose comme : 

```suggestion
    access_modality: list[str] = pydantic.v1.Field(default_factory=list)
```

... et idem pour les autres champs.

Reply:
Si on reçoit nul et qu'on essaie de construire une instance avec `access_modality=None`, ça va coincer, même avec la `default_factory` (`none is not an allowed value (type=type_error.none.not_allowed`)

Ce qui est possible c'est d'ajouter un `root_validator(pre=True)` qui va remplacer `None` par une liste vide. Comme ça, on intercepte la mauvaise valeur et on est tranquille.